src/style.css

```diff
@@ -1 +1,56 @@
-/* add styles here */
+/* #region page */
+html {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+}
+
+/* #endregion */
+
+/* #region stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star {
+  background-image: url(../src/images/star.svg);
+  margin-top: 0;
+}
```

Comment:
The `.stars--0 .stars__star` class has a redundant `background-image` declaration that's the same as in the `.stars__star` class. This could be removed to keep the CSS DRY (Don't Repeat Yourself).